### ISSUE TITLE
chore: add issue and PR template

### ISSUE THREAD

**Issue by Ananya2001-an** [gis-ops/valhalla-app#93]
**Problem**
Currently when contributors are raising a PR to fix some issue they are not linking the issue with the PR which makes it difficult to close the issue on its own when the PR gets merged.

**Solution**
It would be nice to create a template for PR as well as issues so that things like closing issues could be automated and also issues can be well categorized based on bugs, features, docs etc..

**Additional note**
I would like to work on this.

**Comment by nilsnolde** [comment]
Yeah there's some things missing here that I usually do on foss projects, like changelog, issue/pr templates, contributing guide etc. I didn't start this project personally and was dragging my feet on adding those in the past year when I became the de-factor maintainer. 

I'd very much welcome such stuff to make it easier to manage, thanks.

**Comment by Ananya2001-an** [comment]
Great I will raise a PR then.
